### PR TITLE
Update woocommerce-admin links to point to woocommerce repo

### LINKED ISSUE #32516: Update the links of woocommerce-admin in docs and `package.json`
### Describe the solution you'd like

We still have many places with the URL pointing to https://github.com/woocommerce/woocommerce-admin

We should update them to https://github.com/woocommerce/woocommerce.

For example:

- https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/components/package.json#L12-L19

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by chihsuan**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #32516.

- Update the links of woocommerce-admin in docs and package.json to point to https://github.com/woocommerce/woocommerce.
- Removed woocommerce-admin link from contributing guide.

### How to test the changes in this Pull Request:

1. Review the changes and make sure all links are correctly updated.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

docs: Update woocommerce-admin links to point to woocommerce repo

### FOR PR REVIEWER ONLY:

* [x] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -41,9 +41,8 @@ If you have questions about the process to contribute code or want to discuss de
 - Make sure to write good and detailed commit messages (see [this post](https://chris.beams.io/posts/git-commit/) for more on this) and follow all the applicable sections of the pull request template.
 - Please avoid modifying the changelog directly or updating the .pot files. These will be updated by the WooCommerce team.
 
-If you are contributing code to the (Javascript-driven) WooCommerce Admin project or to Gutenberg blocks, note that these are developed in external packages.
+If you are contributing code to the (Javascript-driven) to Gutenberg blocks, note that it's developed in external package.
```
Comment (adrianduffell) [inline .github/CONTRIBUTING.md]:
```suggestion
If you are contributing code to the (Javascript-driven) Gutenberg blocks, note that it's developed in an external package.
```

**Review by adrianduffell — CHANGES_REQUESTED** [review]
This looks great, thanks for taking care of these details 👍 

I've just left a minor suggestion for a wording change.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (chihsuan) [reply .github/CONTRIBUTING.md]: Good catch. Committed the suggestion!

**Review by adrianduffell — APPROVED** [review]

**Comment by adrianduffell** [comment]
LGTM thanks @chihsuan! 🚀

**Pull request merged**